reconcile/templating/validator.py

```diff
@@ -127,7 +127,7 @@ def run(self, dry_run: bool) -> None:
         if diffs:
             for diff in diffs:
                 logging.error(f"template: {diff.template}, test: {diff.test}")
-                logging.debug(diff.diff)
+                logging.error(diff.diff)
```

Comment:
I removed this, cause you can easily leak secrets with this. (usage of vault reader function in jinja)

Reply:
I'll add a comment, to explain this in the code section.

Reply:
I see